### ISSUE TITLE
Using http-request in junit

### ISSUE THREAD

**Issue by mhmdsalem1993** [JetBrains/http-request-in-editor-spec#1]
Do you have any plans so that http-request files can be used inside of junit? if no then whats the parser Class Name in the plugin or even where i can find the plugin itself ?

**Comment by mhmdsalem1993** [comment]
> Would you please elaborate what do you mean? Generate junit xml report based on response tests to execute them on CI?

Yeah exactly maybe for environment testing by using them instead of jmeter
So that maybe not in junit but with a maven plugin.

> The plugin is bundled with IntelliJ products but it's not distributed separately.

Ok , Thanks ! And by the way what an amazing plugin :+1:

**Comment by SvetlanaZem** [comment]
We have plans to implement integration with CI (https://youtrack.jetbrains.com/issue/WI-42002) but currently, we are not going to extend it with performance testing.
